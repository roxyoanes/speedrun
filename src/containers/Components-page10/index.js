export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as About } from "./About";
export { default as Features } from "./Features";
export { default as Screenshots } from "./Screenshots";
export { default as Clients } from "./Clients";
export { default as Packages } from "./Packages";
export { default as Download } from "./Download";
export { default as Footer } from "./Footer";
