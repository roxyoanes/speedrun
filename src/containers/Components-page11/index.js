export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as Services } from "./Services";
export { default as About } from "./About";
export { default as Team } from "./Team";

