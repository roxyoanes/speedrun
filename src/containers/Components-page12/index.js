export { default as Home } from "./Home";
export { default as Navbar } from "./Navbar";
export { default as About } from "./About";
export { default as Skills } from "./Skills";
export { default as Projects } from "./Projects";
export { default as Experience } from "./Experience";

