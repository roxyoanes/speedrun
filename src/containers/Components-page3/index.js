export { default as About } from "./About";
export { default as Education } from "./Education";
export { default as Experience } from "./Experience";
export { default as Interests } from "./Interests";
export { default as Skills } from "./Skills";
export { default as Awards } from "./Awards";
export { default as Navbar } from "./Navbar";
