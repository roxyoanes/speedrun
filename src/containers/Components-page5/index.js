export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as About } from "./About";
export { default as Services } from "./Services";
export { default as Gallery } from "./Gallery";
export { default as Pricing } from "./Pricing";
export { default as Team } from "./Team";
export { default as Clients } from "./Clients";
export { default as Contact } from "./Contact";
export { default as Footer } from "./Footer";
