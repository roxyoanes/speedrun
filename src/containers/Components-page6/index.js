export { default as First } from "./First";
export { default as About } from "./About";
export { default as Services } from "./Services";
export { default as Portfolio } from "./Portfolio";
export { default as Pricing } from "./Pricing";
export { default as Contact } from "./Contact";
