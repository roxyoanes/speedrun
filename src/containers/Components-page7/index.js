export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as About } from "./About";