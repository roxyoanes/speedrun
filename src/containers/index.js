/* eslint-disable import/prefer-default-export */
export { default as Home } from "./Home";

export { default as FirstPage } from "./FirstPage";
export { default as SecondPage } from "./SecondPage";
export { default as ThirdPage } from "./ThirdPage";
