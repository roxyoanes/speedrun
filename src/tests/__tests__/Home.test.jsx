import React from "react";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import { Home } from "../../containers";

it("matches snapshot", () => {
  const wrapper = renderer.create(
    <MemoryRouter>
      <Home />
    </MemoryRouter>,
  );
  expect(wrapper).toMatchSnapshot();
});
