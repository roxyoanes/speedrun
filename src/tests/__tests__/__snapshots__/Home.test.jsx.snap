// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <a
    href="/page-1"
    onClick={[Function]}
  >
    Page1
  </a>
  <a
    href="/page-2"
    onClick={[Function]}
  >
    Page2
  </a>
  <a
    href="/page-3"
    onClick={[Function]}
  >
    Page3
  </a>
  <a
    href="/page-4"
    onClick={[Function]}
  >
    Page4
  </a>
  <a
    href="/page-5"
    onClick={[Function]}
  >
    Page5
  </a>
  <a
    href="/page-6"
    onClick={[Function]}
  >
    Page6
  </a>
</div>
`;
